.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: scala
+dist: trusty
+os:
+  - linux
+scala:
+  - 2.10.6
+jdk:
+  - oraclejdk8
+env:
+  global:
+    - TRAVIS_NODE_VERSION="8"
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+    - $HOME/.coursier/cache
+before_install:
+  # Install Node.js and yarn.
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
nice I will update this on as well: https://github.com/scalameta/metadoc/blob/master/.travis.yml#L23